apps/client/src/dashboard/dashboard.page.tsx

```diff
@@ -0,0 +1,8 @@
+import { useMatches } from 'react-router-dom';
+
+export const DashboardPage = () => {
+  const matches = useMatches();
+  const dashboardId = matches[matches.length - 1]?.params.id;
```

Comment:
it the last match always going to the one holding the params?

Reply:
I plan to remove the dependency on useMatches on the page in a follow-up, but from my observation, all matches hold the params.